tests/unit/data/telstra/telstra2_result.json

```diff
@@ -11,6 +11,6 @@
         "maintenance_id": "987654321-1",
         "start": 1621677600,
         "status": "CONFIRMED",
-        "summary": "Telstra C2C Segment 6 (Singapore - Philippines) Submarine Cable Repair. This is to notify that Telstra will be conducting 2x Repeater R36 & R41A Replacement Work on C2C Segment 6 (Singapore - Philippines due to suspected fauly Laser Diodes to reinstate the End-to-End connectivity of Faulty and Degraded Fiber Pairs and optimize the maximum performance of all Fiber Pairs on this Segment between Singapore and Philippines. Unprotected circuits will be down throughout the repair window. Schedule of this work may change as they are dependent on many outside factors (weather / rough sea / permits/ hostile waters etc) which are beyond Telstra control. Our sincere apologies for any inconvenience caused.\r\n\r\nUpdate #1: Change of schedule  due to availability of  cable repair ship CS Teneo"
+        "summary": "Telstra C2C Segment 6 (Singapore - Philippines) Submarine Cable Repair. This is to notify that Telstra will be conducting 2x Repeater R36 & R41A Replacement Work on C2C Segment 6 (Singapore - Philippines due to suspected fauly Laser Diodes to reinstate the End-to-End connectivity of Faulty and Degraded Fiber Pairs and optimize the maximum performance of all Fiber Pairs on this Segment between Singapore and Philippines. Unprotected circuits will be down throughout the repair window. Schedule of this work may change as they are dependent on many outside factors (weather / rough sea / permits/ hostile waters etc) which are beyond Telstra control. Our sincere apologies for any inconvenience caused.\n\nUpdate #1: Change of schedule  due to availability of  cable repair ship CS Teneo"
```

Comment:
Hmm, okay - is the change from `\r\n` to `\n` throughout intended?

Reply:
with the rewrite via `sys.stdout.write`, these `\r` are getting removed. I guess they were taken when getting the value from the screen, but are not relevant for the test. Do you think necessary to find the right adjustment in the replace mechanism to honour them?